test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] According to the requirements, please make sure that this file is ignored by git. Ignored files shouldn't be present in your remote repository. 🤓 🛠️

Reply:
- [ ] Please kindly implement what @abdulhamiid recommended. As shown below, I still see your `test.md` file , and this is not consistent with the [project requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md):

![image](https://user-images.githubusercontent.com/99715640/211871724-31ad8923-f985-4c15-b0e5-24786014bf1c.png)